.gitignore

```diff
@@ -1 +1,3 @@
 *.code-workspace
+.env
+commit.txt
```

Comment:
nit: what is this file for?

Reply:
it is the commit.txt message file I used for the command `git commit -F commit.txt`

Reply:
deleted commit.txt from .gitignore